src/graph/RadarTrackObjectsNode.cpp

```diff
@@ -88,8 +90,7 @@ void RadarTrackObjectsNode::enqueueExecImpl()
 					return std::abs(distanceHostPtr->at(checkedDetectionId) - distance) <= distanceThreshold &&
 					       std::abs(azimuthHostPtr->at(checkedDetectionId) - azimuth) <= azimuthThreshold &&
 					       std::abs(elevationHostPtr->at(checkedDetectionId) - elevation) <= elevationThreshold &&
-					       (std::isunordered(radialSpeed, radialSpeedHostPtr->at(checkedDetectionId)) ||
-					        std::abs(radialSpeedHostPtr->at(checkedDetectionId) - radialSpeed) <= radialSpeedThreshold);
+					       std::abs(radialSpeedHostPtr->at(checkedDetectionId) - radialSpeed) <= radialSpeedThreshold;
```

Comment:
In my opinon, `std::isunordered` is still necessary here - radial speed comes from detecttion and nothing regarding this topic has changed in this PR.